modeler.js

```diff
@@ -0,0 +1,131 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that creates a new project.
+async function createProject([projectName]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const modelerApiUrl = process.env.MODELER_API_URL;
+
+  // This is the API endpoint to create a new project.
+  const url = `${modelerApiUrl}/projects`;
+
+  // Configure the API call.
+  const options = {
+    method: "POST",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    },
+    data: {
+      // The body contains information about the new project.
+      projectName: projectName
+    }
```

Comment:
```suggestion
      name: projectName
```

The API fields are specified in [the swagger UI](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Projects/createProject).